accepted/package-importer.d.ts.md

```diff
@@ -554,8 +554,8 @@ potential subpaths, resolving for partials and file extensions.
 
   * Add `subpath` to `paths`.
 
-* Otherwise, add `subpath` + `.scss`, `subpath` + `.sass`, and `subpath` +
-  `.css` to `paths`.
+* Otherwise, add `subpath`, `subpath` + `.scss`, `subpath` + `.sass`, and
```

Comment:
I think we should make it explicit to only add the extensions when it's not specified. If it already has an extension (.scss, .sass, or .css), we should not try add another.

Reply:
This is cut off in this snippet, but the preceding lines contain an "if it has an extension", and this is the else.

```
* If `subpath` ends in `.scss`, `.sass`, or `.css`:

  * Add `subpath` to `paths`.

* Otherwise...
```